apps/librelingo_yaml_loader/tests/test_yaml_loader.py

```diff
@@ -40,12 +35,12 @@ def create_patch(self, module):
         patcher = patch(module)
         self.addCleanup(patcher.stop)
         mock = patcher.start()
-        mock.return_value = random.randint(0, 1000)
+        mock.return_value = fakes.fake_value().id
```

Comment:
why `.id`?

Reply:
Before `mock.return_value` was an `int` not a `namedtuple("FakeValue", ["id"])`.

Reply:
What do you think about adding a function `number()` to [`fakes.py`](https://github.com/kantord/LibreLingo/blob/37a04d004cbc341f0ffe8b256c1f5a6a168539ad/apps/librelingo_fakes/librelingo_fakes/fakes.py) and
- apply it in [`fake_value()`](https://github.com/kantord/LibreLingo/blob/37a04d004cbc341f0ffe8b256c1f5a6a168539ad/apps/librelingo_fakes/librelingo_fakes/fakes.py#L270),
- use it instead of `fakes.fake_value().id`.

Reply:
yes this makes a lot of sense, @vil02 !!